lib/serverengine/socket_manager_win.rb

```diff
@@ -15,12 +15,14 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 #
+require 'socket'
+require 'ipaddr'
```

Comment:
I think it is better (more conservative) that `socket_manager_unix.rb` also requires `ipaddr` because it's a possible pitfall for users that `IPAddr` becomes automatically available on Windows (and some test may pass on Windows) but it's not on Linux (and some production code may fail). Although it's impossible to abstract everything, requiring `ipaddr` seems like low-hanging fruit.